etc/config/android-java.amazon.properties

```diff
@@ -21,16 +22,22 @@ compiler.java-d8-8172.exe=/opt/compiler-explorer/r8-8.1.72/r8-8.1.72.jar
 compiler.java-d8-8156.name=d8 8.1.56
 compiler.java-d8-8156.exe=/opt/compiler-explorer/r8-8.1.56/r8-8.1.56.jar
 
-group.dex2oat.compilers=java-dex2oat-latest
+group.dex2oat.compilers=java-dex2oat-latest:java-dex2oat-3411
 group.dex2oat.groupName=ART
 group.dex2oat.compilerType=dex2oat
+group.dex2oat.isSemVer=true
+group.dex2oat.d8Id=java-d8-8242
+group.dex2oat.d8Path=/opt/compiler-explorer/r8-8.2.42/r8-8.2.42.jar
```

Comment:
This looks ok at first glance, but I do believe this will get you into trouble when you want to update this version for -latest, because you really want the fixed version to also use a fixed version in its dependencies.

I would suggest keeping it at the compiler level.